src/containers/CampaignList/components/CampaignHeader.tsx

```diff
@@ -0,0 +1,99 @@
+import { Divider, Typography } from "@material-ui/core";
+import Chip from "@material-ui/core/Chip";
+import { green, orange } from "@material-ui/core/colors";
+import CheckCircleOutlineRoundedIcon from "@material-ui/icons/CheckCircleOutlineRounded";
+import WarningRoundedIcon from "@material-ui/icons/WarningRounded";
+import React from "react";
+
+type IconMap = {
+  [key: string]: JSX.Element;
+};
+
+const CHIP_ICONS: IconMap = {
+  "Not Started": <WarningRoundedIcon style={{ color: orange[300] }} />,
+  "Unassigned Contacts": <WarningRoundedIcon style={{ color: orange[300] }} />,
+  "Unsent Initial Messages": (
+    <WarningRoundedIcon style={{ color: orange[300] }} />
+  ),
+  "All Contacts Assigned": (
+    <CheckCircleOutlineRoundedIcon style={{ color: green[500] }} />
+  ),
+  "All Initials Sent": (
+    <CheckCircleOutlineRoundedIcon style={{ color: green[500] }} />
+  )
+};
+
+const inlineStyles = {
+  wrapper: {
+    display: "flex",
+    flexWrap: "wrap",
+    alignItems: "center"
+  },
+  chip: {
+    margin: "4px",
+    padding: "4px",
+    borderRadius: "2px"
+  }
+};
+
+export type Tag = {
+  title: string;
+  backgroundColor?: string;
+  color?: string;
+};
+
+interface CampaignHeaderProps {
+  campaignTitle: string;
+  campaignId: string;
+  tags: Tag[];
+  onClick: () => void;
+}
+
+const CampaignHeader = (props: CampaignHeaderProps) => {
+  const { campaignTitle, campaignId, tags, onClick } = props;
+  return (
+    <div style={inlineStyles.wrapper}>
+      <Typography
+        variant="h5"
```

Comment:
Suggestion: Size this down to not use as much space - see notion doc https://rewiredcoop.slack.com/archives/C02KUGE68UX/p1690904317810059?thread_ts=1690808489.583299&cid=C02KUGE68UX
```suggestion
        variant="h6"
```